packages/block-library/src/navigation/index.php

```diff
@@ -561,13 +567,30 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 	$has_old_responsive_attribute = ! empty( $attributes['isResponsive'] ) && $attributes['isResponsive'];
 	$is_responsive_menu           = isset( $attributes['overlayMenu'] ) && 'never' !== $attributes['overlayMenu'] || $has_old_responsive_attribute;
 	$should_load_view_script      = ! wp_script_is( 'wp-block-navigation-view' ) && ( $is_responsive_menu || $attributes['openSubmenusOnClick'] || $attributes['showSubmenuIcon'] );
-	if ( $should_load_view_script ) {
-		wp_enqueue_script( 'wp-block-navigation-view' );
-	}
 
-	$should_load_modal_view_script = isset( $attributes['overlayMenu'] ) && 'never' !== $attributes['overlayMenu'];
-	if ( $should_load_modal_view_script ) {
-		wp_enqueue_script( 'wp-block-navigation-view-modal' );
+	if ( $should_load_view_script ) {
+		if ( $is_interactivity_api_enabled ) {
+			wp_enqueue_script(
+				'interactivity-navigation',
+				plugins_url( '../interactive-blocks/navigation.min.js', __FILE__ ),
```

Comment:
There is also `__DIR__` constant that might work here better:
```suggestion
				plugins_url( './interactive-blocks/navigation.min.js', __DIR__ ),
```

Reply:
I just changed it 🙂